packages/cli/src/databases/migrations/common/1695128658538-AddWorkflowMetadata.ts

```diff
@@ -0,0 +1,13 @@
+import type { MigrationContext, ReversibleMigration } from '@db/types';
+
+export class AddWorkflowMetadata1695128658538 implements ReversibleMigration {
+	transaction = false as const;
```

Comment:
this needs to be set only if the DB is sqlite.

Reply:
ah, I see. thanks for spotting updated now.